libdenaro/include/models/group.h

```diff
@@ -0,0 +1,107 @@
+#ifndef GROUP_H
+#define GROUP_H
+
+#include "color.h"
+#include <string>
```

Comment:
I like to order it with system includes first then local, so string first then color.h

Reply:
ok. will do